api/design/status.go

```diff
@@ -1,3 +1,17 @@
+// Copyright © 2020 The Tekton Authors.
```

Comment:
Should this change come under this commit ?

Reply:
yes, for all new files